assets/js/blocks/cart/edit.js

```diff
@@ -111,6 +112,7 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
 				/>
 			</PanelBody>
 			<CartCheckoutFeedbackPrompt />
+			<CartSidebarCompatibilityNotice />
```

Comment:
We should place the alert under the `Name & description` panel before the Block's settings panels (i.e., Let's move it to `line 70`).